py_zipkin/_encoding_helpers.py

```diff
@@ -150,6 +160,32 @@ def build_v1_span(self):
             sa_endpoint=self.sa_endpoint,
         )
 
+    def build_v2_span(self):
+        """Builds and returns a V2 Span.
+
+        :return: newly generated _V2Span
+        :rtype: _V2Span
+        """
+        remote_endpoint = None
+        if self.sa_endpoint:
+            remote_endpoint = self.sa_endpoint
+
+        return _V2Span(
+            trace_id=self.trace_id,
+            name=self.name,
+            parent_id=self.parent_id,
+            id=self.span_id,
+            kind=self.kind,
+            timestamp=self.timestamp if self.report_timestamp else None,
+            duration=self.duration if self.report_timestamp else None,
+            debug=False,
```

Comment:
not sure if this implies serialization, but both of the bools should be omitted if false. Also None values like parent timestamp duration etc should also be ommitted.